test/run/ch-run_misc.bats

```diff
@@ -33,6 +65,13 @@ EOF
     ch-run -w "$ch_timg" rm write
 }
 
+@test 'run image by name' {
+    [[ $CH_TEST_BUILDER = ch-image ]] || skip 'ch-image only'
+    run ch-run 00_tiny -- echo foo
```

Comment:
Instead of `echo foo` and checking the output, just use `true` and check if it succeeds.